firmware/baseband/sd_over_usb/usb_descriptor.c

```diff
@@ -25,10 +25,10 @@
 #include "usb_type.h"
 #include "usb_descriptor.h"
 
-#define USB_VENDOR_ID (0x1D50)
+#define USB_VENDOR_ID (0x0781) /* SanDisk Corp. */
 
 #ifdef HACKRF_ONE
```

Comment:
I guess these ifdef's are kinda pointless at this point :)

Reply:
They are. There many more in that file. Since it is a copy from the original hackrf, I wanted as little changes as possible to later see what changes are relevant.